sled-agent/src/rack_setup/service.rs

```diff
@@ -783,6 +783,10 @@ impl ServiceInner {
             // the need for unencrypted communication.
             certs: vec![],
             internal_dns_zone_config: d2n_params(&service_plan.dns_config),
+            // TODO This eventually needs to come from the person setting up the
+            // system.
+            external_dns_zone_name:
+                internal_dns::names::DNS_ZONE_EXTERNAL_TESTING.to_owned(),
```

Comment:
Could we add this to the config (`RackInitializeRequest`) instead of hardcoding it?

Reply:
I think that is probably the next step.  Do you mind if I do that in a follow-on PR?  (Previous to this PR, it was hardcoded in Nexus.  This at least plumbs the value from here the rest of the way, so I think it's an improvement.)  I want to take some time to better understand that path.